proca/lib/proca_web/schema/action_types.ex

```diff
@@ -42,6 +77,39 @@ defmodule ProcaWeb.Schema.ActionTypes do
 
       resolve(&Resolvers.ExportActions.export_actions/3)
     end
+
+    @desc """
+    Get contacts collected by org, optionally filtered by campaign
+    """
+    field :contacts, non_null(list_of(:action)) do
```

Comment:
Honestly, it should just be an arg instead of a duplicate query?

Reply:
I imagine they will evolve and diverge in the future, better to keep them separate